package.json

```diff
@@ -26,6 +26,7 @@
     "discord.js": "^14.18.0",
     "express": "^5.0.1",
     "isomorphic-git": "^1.25.6",
-    "ts-node": "^10.9.2"
+    "ts-node": "^10.9.2",
+    "ws": "^8.18.1"
```

Comment:
Are we not using node 22 anymore?

Reply:
No, native ws API doesn't support headers